### PR TITLE
Enable Chat backup, Mentions and Read Receipt in release.

### PR DESCRIPTION
**Pull request by bmarty**
Also change default for debug to enabled, but keep `isFinished = false` for now, so that they are still disableable.

 
## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Technical
- [x] Other : enable all the features!

## Tests

- Step 1
- Step 2
- Step ...

## Tested devices

- [ ] Physical
- [ ] Emulator
- OS version(s):

## Checklist

- [ ] Changes have been tested on an Android device or Android emulator with API 23
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
- [ ] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [ ] You've made a self review of your PR

### CODE REVIEW

**Comment by bmarty** [comment]
I need to fix the test

**Comment by bmarty** [comment]
Note: I ignore failing test in release `present - logout then confirm` for now

File: features/messages/impl/src/test/kotlin/io/element/android/features/messages/impl/textcomposer/MessageComposerPresenterTest.kt [hunk 1]
```diff
@@ -929,6 +904,12 @@ class MessageComposerPresenterTest {
         currentSessionIdHolder = CurrentSessionIdHolder(FakeMatrixClient(A_SESSION_ID)),
         permissionsPresenterFactory = FakePermissionsPresenterFactory(permissionPresenter),
     )
+
+    private suspend fun <T> ReceiveTurbine<T>.awaitFirstItem(): T {
+        // Skip 2 item if Mentions feature is enabled, else 1
+        skipItems(if (FeatureFlags.Mentions.defaultValue) 2 else 1)
+        return awaitItem()
+    }
```
Comment (jmartinesp) [inline features/messages/impl/src/test/kotlin/io/element/android/features/messages/impl/textcomposer/MessageComposerPresenterTest.kt:912]: Clever, thanks!

**Review by jmartinesp — APPROVED** [review]
Thank you for all the work figuring out a fix for the broken tests

**Pull request merged**